src/main/java/com/snowplowanalytics/snowplow/tracker/Utils.java

```diff
@@ -77,7 +77,8 @@ public static boolean isValidUrl(String url) {
             new URL(url).toURI();
             return true;
         } catch (Exception e) {
-            LOGGER.error("URI {} is not valid: {}", url, e.getMessage());
+            LOGGER.debug("URI {} is not valid: {}", url, e.getMessage());
```

Comment:
nit: I wonder if we should swap the order of these logs around in this file? If you're in debug mode, I think it'd be nice to get the error printed first, then see the debug info (at least I think thats how my brain would expect to read it when looking at logs).

Reply:
yeah that makes sense